www/widgets/DemoExampleMaker.js

```diff
@@ -291,6 +310,47 @@ class DemoExampleMaker extends Widget {
     this.$('[name="dem-url"]').style.display = 'block'
   }
 
+  _uploadJSON (ex) {
+    for (let step = this._data.steps.length; step >= 1; step--) {
+      this._updateStep(step, 'remove')
+    }
+    this._data = {
+      name: ex.name,
+      tags: ex.tags || null,
+      toc: ex.toc || true,
```

Comment:
if the example doesn't have a `toc` than that means it's likely not annotated, so while i think the default value for a newly created example with annotations should be `true`, in this case (the default value for an old example we're uploading that's missing a `toc`) should be `false` (otherwise it opens the toc even though there aren't any steps to really go through)